boggle.js

```diff
@@ -1,7 +1,7 @@
 function newBoggleBoard() {
 
   let boggleBoard = [];
-  for(i=0;i<=4;i++)
+  for(i=0;i<4;i++)
   {
     boggleBoard.push(['_', '_', '_', '_'])
```

Comment:
This is fine, but I don't really understand why you changed it from the skeleton code.